docs/docs/guides/exporting-logs.md

```diff
@@ -0,0 +1,206 @@
+
+Start an Elasticsearch container with the following command:
+
+```bash
+docker run -d --name es01 --net bridge -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" elasticsearch:8.17.3
+```
+
+Note, the network must be set to `bridge` for the logs aggregator to be able to connect to Elasticsearch.
+
+Start a Kibana container with following command:
+
+```bash
+docker run -d --name kb --net bridge -p 5601:5601 kibana:8.17.3
+```
+
+Generate an Elasticsearch enrollment token with the following command:
+
+```bash
+docker exec -it es01 /usr/share/elasticsearch/bin/elasticsearch-create-enrollment-token -s kibana
+```
+
+Access Kibana at `http://localhost:5601` and paste the enrollment token generated by the command above. Subsequently, Kibana
+will ask you for a 6-digit verification code. To view this code, access Kibana container logs:
+
+```bash
+docker logs kb
+```
+
+Next, Kibana will ask you to sign in. The username for the local Elasticsearch instance is `elastic`, and you can generate
+the password with:
+
+```bash
+docker exec -it es01 /usr/share/elasticsearch/bin/elasticsearch-reset-password -u elastic
+```
+
+You have now successfully started an Elasticsearch instance.
+
+## Configuring Kurtosis to send logs to Elasticsearch
+
+Locate your Kurtosis configuration file with the following command:
+
+```bash
+kurtosis config path
+```
+
+Determine the IP address of your Elasticsearch container by running the following command:
+
+```bash
+docker inspect -f '{{.NetworkSettings.Networks.bridge.IPAddress}}' es01
+```
+
+Open the configuration file and paste the following content, replacing `<PASSWORD>` with the Elasticsearch password generated
+in the previous section and `<ELASTICSEARCH_IP_ADDRESS>` with the IP address of your Elasticsearch container:
+
+```yaml
+config-version: 3
+should-send-metrics: true
+kurtosis-clusters:
+  docker:
+    type: "docker"
+    logs-aggregator: 
+      sinks: 
+        elasticsearch:
+          type: "elasticsearch"
+          bulk: 
+            index: "kt-{{ enclave_uuid }}-{{ service_name }}"
+          auth:
+            strategy: "basic"
+            user: "elastic"
+            password: "<PASSWORD>"
+          tls:
+            verify_certificate: false
+          endpoints:
+            - "https://<ELASTICSEARCH_IP_ADDRESS>:9200"
+```
+:::info
+`config-version` must be set to 3 for logs aggregator configurations to apply
+:::
+
+:::danger
+`tls.verify_certificate` should not be disabled outside of testing!
+:::
+
+Finally, restart Kurtosis engine to apply the changes:
+
+```bash
+kurtosis engine restart
+```
+
+## Verify logs delivery
+
+To verify that Kurtosis are actually exporting logs to Elasticsearch, start by running a package. In this guide, we use
+`ethpandaops/ethereum-package`:
+
+```bash
+kurtosis run github.com/ethpandaops/ethereum-package
+```
+
+Once the package has finished execution, go to  `http://localhost:5601/app/enterprise_search/content/search_indices`.
```

Comment:
optional but would be great to have a picture or visual of what users might expect both as a "heres what it should look like" and a "hooray, it works" to wrap up the guide.

Reply:
Yes can do